packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/react",
-  "version": "2.8.13",
+  "version": "2.8.14-alpha.1",
```

Comment:
Perfect! 
Please also bump the version of `@web3-onboard/core` on line 65 below?

Reply:
Thank you! I've also bumped the version in https://github.com/blocknative/web3-onboard/pull/2057/commits/fc3c8316025f6b486bc13e83ea11713e28bb3f44